tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -39,7 +39,8 @@ public sealed class BuildUpToDateCheckTests : BuildUpToDateCheckTestBase, IDispo
         private bool _isTaskQueueEmpty = true;
         private bool _isFastUpToDateCheckEnabledInSettings = true;
         private bool _isBuildAccelerationEnabledInSettings;
-        private bool? _isBuildAccelerationEnabled;
+        private bool? _isBuildAccelerationEnabledInProject;
```

Comment:
I renamed this, which increased the diff size here unfortunately. But the field no longer specified whether BA is _enabled_, only whether the `AccelerateBuildsInVisualStudio` property is present and with what value. We have added other ways of controlling whether BA is enabled (feature flag and incompatible package references) so the old name became misleading.